src/app/plugins/plugin_detect_balls.h

```diff
@@ -74,11 +74,11 @@ friend class PluginDetectBalls;
 
   _settings=new VarList("Ball Detection");
 
-  _settings->addChild(_max_balls = new VarInt("Max Ball Count",10));
+  _settings->addChild(_max_balls = new VarInt("Max Ball Count",20));
   _settings->addChild(_color_label = new VarString("Ball Color","Orange"));
 
   _settings->addChild(_filter_general = new VarList("Ball Properties"));
-    _filter_general->addChild(_ball_z_height = new VarDouble("Ball Z-Height", 30.0));
+    _filter_general->addChild(_ball_z_height = new VarDouble("Ball Z-Height", 21.5));
```

Comment:
I'm still not sure if the 30mm were intended.
The ball has a radius of 21.5mm. When looking from above, I see the full shape of the ball, which is at a height of one radius.

Reply:
I've experimented with this value on my field and it turned out that 30mm actually seems to be good. I still do not fully understand, why, though.

Reply:
Yes, we have consistently observed the same, which is why it is so. Can you add a comment to the effect? Let's prevent people in the future try to set it back. ;-)
I suspect it is partially due to the ball sinking into the carpet, and partially due to bias in the geometry calibrator.

Reply:
done.